plans/README.md

```
@@ -138,10 +138,375 @@ Recommended order is by priority then leverage. Plans are independent unless "De
+> **型エラーで露見する**のに対し、リレーションフィルタは一意性を要求しないため
+> **コンパイルが通ったまま実行時に別ノードへ一致し得る**。「壊れても気づけない経路」が
+> 存在することが、親内一意（`@@unique([parentId, url])`）を却下した決め手になった
+> （ADR-006 Option 4）。副次的に、この決定によって経路 B は **067 で書き換え不要**になっている。
+>
+> **本 spike が主張しないこと**: (1) 性能ベンチマークは取っていない —— 方式選定はクエリ
+> **形状**と規模見積（カテゴリは O(10^2〜10^3) 行）に基づく。(2) ツリー UI（DnD エディタ）と
+> 参照タクソノミー 20 部門の投入は範囲外（068 の Out of scope に明記）。(3) `deleteCategory` の
+> 「無効化 + 付け替え」化は別プランで起票する。
+>
+> **副次的な整備**: ADR 一覧の索引に未登録だった **004 / 005** も併せて追加した（`ae2c96d0`）。
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)